tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/partners_block.html

```diff
@@ -1,12 +1,16 @@
 {% load wagtailimages_tags %}
 
-{% if value.title %}
-    <h2 class="partners__title heading--two">{{ value.title }}</h2>
-{% endif %}
+<div class="partners streamfield__partners">
```

Comment:
I've just merged the `grid__` refactor so this will need updating when you rebase.

Reply:
thanks! Have updated